.github/workflows/test.yml

```diff
@@ -51,11 +51,14 @@ jobs:
         cargo install --version 0.9.1 wasm-pack
     - name: Build
       run: cargo build --verbose $TARGET
+    - name: Disable benchmarking dependencies
+      if: matrix.mode != 'native' || matrix.rust-version == '1.33.0'
+      run: "mv Cargo.toml Cargo.toml.bak && sed '/ # benchmark only$/d' Cargo.toml.bak >Cargo.toml"
     - name: Run tests
       if: matrix.mode == 'native'
       run: cargo test --verbose
     - name: Run benchmarks
-      if: matrix.mode == 'native'
+      if: matrix.mode == 'native' && matrix.rust-version != '1.33.0'
```

Comment:
Same as above: Maybe check for `(matrix.rust-version != 'stable' && matrix.rust-version != 'nightly')` here?